doc/install.rst

```diff
@@ -28,15 +28,13 @@ or ``conda``::
 
     conda install scikit-learn
 
-**We don't recommend installing scipy or numpy using pip on linux**,
-as this will involve a lengthy build-process with many dependencies.
-Without careful configuration, building numpy yourself can lead to an installation
-that is much slower than it should be. 
-If you are using Linux, consider using your package manager to install
-scikit-learn. It is usually the easiest way, but might not provide the newest
-version.
-If you haven't already installed numpy and scipy and can't install them via
-your operation system, it is recommended to use a third party distribution.
+If you have not installed NumPy or SciPy yet, you can also install these using
+conda or pip. When using pip, please ensure that *binary wheels* are used,
+and NumPy and SciPy are not recompiled from source, which can happen when using
+particular configurations of operating system and hardware (such as Linux on
+a Raspberry Pi). We don't recommend installing numpy or scipy from source.
```

Comment:
Instead of the frightening "We don't recommend installing numpy or scipy from source." I would rather be slightly more specific:

Building numpy and scipy from source can be complex (especially on Windows) and requires careful configuration to ensure that they link against an optimized implementation of linear algebra routines.